dashboard/modules/reporter/reporter_agent.py

```diff
@@ -98,7 +103,16 @@ def jsonify_asdict(o):
     "raylet_cpu": Gauge("raylet_cpu", "CPU usage of the raylet on a node.",
                         "percentage", ["ip", "pid"]),
     "raylet_mem": Gauge("raylet_mem", "Memory usage of the raylet on a node",
-                        "mb", ["ip", "pid"])
+                        "mb", ["ip", "pid"]),
+    "cluster_active_nodes": Gauge("cluster_active_nodes",
+                                  "Active nodes on the cluster", "count",
+                                  ["node_type"]),
+    "cluster_failed_nodes": Gauge("cluster_failed_nodes",
+                                  "Failed nodes on the cluster", "count",
+                                  []),
```

Comment:
don't need the list here

Reply:
am now reporting node type, so this is needed